source/_integrations/fronius.markdown

```diff
@@ -22,14 +22,14 @@ To enable this sensor, add the following lines to your `configuration.yaml` file
 ```yaml
 sensor:
   - platform: fronius
-    resource: FRONIUS_URL
+    resource: FRONIUS_LOCAL_API_URL
     monitored_conditions:
     - sensor_type: inverter
 ```
 
 {% configuration %}
 resource:
-  description: "The URL of the Fronius device (e.g., `http://192.0.2.0` or `http://fronius.local`)"
+  description: "The URL of the API on the Fronius datamanager card (e.g., `http://192.0.2.0` or `http://fronius.local`)"
```

Comment:
Is the API different from regular device access? As in, as it specifically mentions it, is that needed?

Reply:
Clarified that only root URL is needed.